src/Text/LLVM/PP.hs

```diff
@@ -45,29 +85,43 @@ data Config = Config { cfgLoadImplicitType :: Bool
                        -- instruction is implied.
 
                      , cfgUseDILocation :: Bool
+
+                     , cfgVer :: LLVMVer
```

Comment:
I'd be willing to bet that all of the fields above this one (`cfgLoadImplicitType`, `cfgGEPImplicitType`, and `cfgUseDILocation`) can be subsumed by checking `cfgVer`. I'm not sure if you were planning to do that in a separate PR, however.

Reply:
Yup, that's the plan.